scripts/install_third_party_libs.py

```diff
@@ -123,8 +123,8 @@ def pip_install(package, version, install_path):
         raise Exception
 
     subprocess.check_call([
-        'pip', 'install', '%s==%s' % (package, version), '--target',
-        install_path])
+        'python', '-m', 'pip', 'install', '%s==%s' % (package, version),
```

Comment:
While you're at it ... perhaps worth weighing in on #8123?

Reply:
So the reason behind this change was so that pip will use the correct python version, I'll note that in the issue.

Reply:
And python will use the correct pip version.

Reply:
How about also using `sys.executable` instead of `python` as this StackOverflow post suggests? Seems like this would also prevent any issues with `python` being linked to Python 2 vs Python 3.

https://stackoverflow.com/a/50255019

Reply:
Ah, that's a good one, I'll try that!